docs/docs/en/contribute/development-environment-setup.md

```diff
@@ -181,7 +181,14 @@ Following steps will guide how to start the DolphinScheduler backend service
 
 - File change
 
-  - If you use MySQL as your metadata database, you need to modify `dolphinscheduler/pom.xml` and change the `scope` of the `mysql-connector-java` dependency to `compile`. This step is not necessary to use PostgreSQL
+  - If you use MySQL as meta-database, you need to add `mysql-connector-java` dependency in `pom.xml` file which is under `dolphinscheduler-api` and `dolphinscheduler-master` modules, it is not needed if you use PostgreSQL, the detailed configuration is as follows:
+  ```xml
+    <dependency>
+      <groupId>mysql</groupId>
+      <artifactId>mysql-connector-java</artifactId>
+      <scope>compile</scope>
+    </dependency>
+    ```
```

Comment:
Hi @QuakeWang currently we can simple comment the mysql dependence scope in `pom.xml` in module `dolphinscheduler-bom`

```xml
<dependency>
    <groupId>mysql</groupId>
    <artifactId>mysql-connector-java</artifactId>
    <version>${mysql-connector.version}</version>
    <scope>test</scope>
</dependency>
```

Reply:
> Hi @QuakeWang currently we can simple comment the mysql dependence scope in `pom.xml` in module `dolphinscheduler-bom`
> 
> ```
> <dependency>
>     <groupId>mysql</groupId>
>     <artifactId>mysql-connector-java</artifactId>
>     <version>${mysql-connector.version}</version>
>     <scope>test</scope>
> </dependency>
> ```

Does this mean that `mysql-connector` will work just by changing the pom file which is under `dolphinscheduler-bom`?